src/presentation/pages/Note.vue

```diff
@@ -77,6 +85,19 @@ function createChildNote(): void {
   router.push(`/note/${props.id}/new`);
 }
 
+/**
+ * Unlink note from parent
+ */
+function unlinkButton(): void {
+  if (props.id === null) {
+    throw new Error('Note is Empty');
+  }
+
+  parentNote.value = undefined;
```

Comment:
its better to move it to the useNote service and place after the domain service call to be sure that note is really unlinked